packages/core/src/typings/poll.ts

```diff
@@ -0,0 +1,69 @@
+import type { PollLayoutType } from "#enums";
+import type { Emoji } from "./emoji.js";
+import type { User } from "./user.js";
+
+export declare namespace Poll {
+    /**
+     * @see {@link https://discord.com/developers/docs/resources/poll#poll-object-poll-object-structure}
+     */
+    export interface Structure {
+        question: MediaStructure;
+        answers: Array<AnswerStructure>;
+        /** ISO8601 Timestamp */
+        expiry: string | null;
+        allow_multiselect: boolean;
+        layout_type: PollLayoutType;
+        results?: ResultStructure;
+    }
+
+    /**
+     * @see {@link https://discord.com/developers/docs/resources/poll#poll-create-request-object-poll-create-request-object-structure}
+     */
+    export interface CreateStructure {
```

Comment:
I think this should be a bit better for this case:

But im just nitpicking at this point

```ts
export export interface CreateStructure extends Omit<Structure, "expiry"> {
    duration: number
}

Reply:
I don't think this is a good thing since there are also differences with results & layout type I would have to do
```ts
export interface CreateStructure extends Omit<Structure, "expiry" | "results" | "layout_type"> {
    duration: number;
    layout_type?: PollLayoutType;
}
```

since `layout_type` is optional in create structure

Reply:
Oh, i did miss the optional part my bad.
Lets leave it like it is then